src/revChatGPT/__main__.py

```diff
@@ -163,6 +163,9 @@ def main():
                     print("Something went wrong!")
                     print(exc)
                     continue
+    except KeyboardInterrupt:
+        print("\nGoodbye!")
+        exit()
```

Comment:
Use `sys.exit()`

Reply:
There are 3 more usages of `exit()` in `__main__.py`, that's thy I just changed the import section to
`from sys import argv, exit`.